ReactiveObjC.xcodeproj/project.pbxproj

```diff
@@ -2823,7 +2824,7 @@
 				CURRENT_PROJECT_VERSION = 1;
 				ENABLE_TESTABILITY = YES;
 				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
-				MACOSX_DEPLOYMENT_TARGET = 10.9;
+				MACOSX_DEPLOYMENT_TARGET = 10.10;
```

Comment:
Can you change the deployment target of the test target only? The actual framework should be able to stay deployable to 10.9.